### PR TITLE
Word-count using std::string.

### PR DESCRIPTION
**Pull request by tremmelg**
Needs #12

### CODE REVIEW

File: examples/tests/word_count.cpp [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+#include <gstorm.h>
+#include <range/v3/all.hpp>
+
+#include "experimental.h"
+
+struct String : public testing::Test {};
+
+// determines whether the character is alphabetical
+bool is_alpha(const char c) { return (c >= 'A' && c <= 'z'); }
```
Comment (Ruyk) [inline examples/tests/word_count.cpp:14]: is there any particular reason std::is_alpha cannot be used here?

**Review by Ruyk — APPROVED** [review]
Looks good, only minor comment

File: examples/tests/word_count.cpp (hunk 1 above)
Comment (tremmelg) [reply examples/tests/word_count.cpp:14]: It won't run on the GPU.
```
warning: [Computecpp:CC0034]: Function isalpha is undefined but referenced on the device and the associated
      kernels may fail to build or execute at run time [-Wsycl-undef-func]
```

Comment (Ruyk) [reply examples/tests/word_count.cpp:14]: Can you create an issue for ComputeCpp to identify this builtin? Thanks

Comment (tremmelg) [reply examples/tests/word_count.cpp:14]: Done.

**Pull request merged**